packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -126,7 +126,22 @@ export const messages = {
   force_stop_apex_debug_logging:
     'SFDX: Turn Off Apex Debug Log for Replay Debugger',
   isv_debug_bootstrap_step1_create_project:
-    'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
-  isv_debug_bootstrap_step1_configure_project:
-    'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project'
+    'SFDX: ISV Debugger Setup, Step 1 of 7: Create Project',
+  isv_debug_bootstrap_step2_configure_project:
+    'SFDX: ISV Debugger Setup, Step 2 of 7: Configure Project',
+  isv_debug_bootstrap_step3_retrieve_org_source:
+    'SFDX: ISV Debugger Setup, Step 3 of 7: Retrieving unpackaged Apex Source',
+  isv_debug_bootstrap_step4_convert_org_source:
+    'SFDX: ISV Debugger Setup, Step 4 of 7: Converting unpackaged Apex Source',
+  isv_debug_bootstrap_step5_list_installed_packages:
+    'SFDX: ISV Debugger Setup, Step 5 of 7: Quering for installed Packages',
+  isv_debug_bootstrap_step6_retrieve_packages_source:
+    'SFDX: ISV Debugger Setup, Step 6 of 7: Retrieving Packages',
+  isv_debug_bootstrap_step7_convert_package_source:
+    'SFDX: ISV Debugger Setup, Step 7 of 7: Converting Package: %s',
+
+  error_creating_packagexml: 'Error creating package.xml: %s',
+  error_extracting_org_source: 'Error extracting downloaded Apex source: %s',
+  error_extracting_packages: 'Error extracting packages: %s',
+  error_updateing_sfdx_project: 'Error updating sfdx-project.json: %s'
```

Comment:
Typo with `updateing`

Reply:
@ruthemmanuelle Can you also review those error messages? They show up in VSCode as errors as a notification.

![2018-02-21_10-19-02](https://user-images.githubusercontent.com/191472/36472029-bf47d564-16f0-11e8-9d4a-f87916dcad63.png)

Note the shiny new notifications in the Insiders build.

Reply:
These look fine (aside from the typo Jonathan pointed out), but I wonder what the '%s'es will be replaced with. Will the error messages provide any guidance on how to recover from the errors?

Reply:
@ruthemmanuelle I'll fix the type. The error messages typically come from a 3rd party (eg., the OS) and they would help analysing/troubleshooting the problem (eg., hard disc running out of space, unable to extract zip archive, etc.).